### PR TITLE
PLF-8339 : Wrong generated URL when clicking on space which names are already edited when applying filters from leftNavigationMenu

### PR DESCRIPTION
**Pull request by AzmiTouil**
current behavior : The URL contain new_space_name/new_space_name

=> The URL must contain Old_space_name/new_space_name

### CODE REVIEW

File: component/common/src/main/java/org/exoplatform/platform/common/space/rest/SpaceRestServiceImpl.java [hunk 1]
```diff
@@ -116,7 +116,7 @@ private void addSpaceWithFieldsToList(Space space, String fields, List<Object> s
           baseSpaceURL.append("/");
           baseSpaceURL.append(permanentSpaceName) ;
       } else {
-          baseSpaceURL.append(space.getPrettyName()) ;
+          baseSpaceURL.append(permanentSpaceName) ;
```
Comment (ahamdi) [inline component/common/src/main/java/org/exoplatform/platform/common/space/rest/SpaceRestServiceImpl.java]: We can replace all the If/Else block with 
`baseSpaceURL.append(permanentSpaceName) ;
 baseSpaceURL.append("/");
 baseSpaceURL.append(space.getPrettyName()) ;`

Comment (AzmiTouil) [reply component/common/src/main/java/org/exoplatform/platform/common/space/rest/SpaceRestServiceImpl.java]: Okay

Comment (AzmiTouil) [reply component/common/src/main/java/org/exoplatform/platform/common/space/rest/SpaceRestServiceImpl.java]: Done .

File: component/common/src/main/java/org/exoplatform/platform/common/space/rest/SpaceRestServiceImpl.java [hunk 2]
```diff
@@ -109,17 +109,12 @@ private void addSpaceWithFieldsToList(Space space, String fields, List<Object> s
...
-          baseSpaceURL.append("/");
-          baseSpaceURL.append(permanentSpaceName) ;
-      } else {
-          baseSpaceURL.append(space.getPrettyName()) ;
-          baseSpaceURL.append("/");
-          baseSpaceURL.append(space.getPrettyName()) ;
-      }
+        StringBuffer baseSpaceURL = new StringBuffer();
+        baseSpaceURL.append(PortalContainer.getCurrentPortalContainerName() + "/g/:spaces:");
+
```
Comment (ahamdi) [inline component/common/src/main/java/org/exoplatform/platform/common/space/rest/SpaceRestServiceImpl.java]: Fix indentation please
```suggestion
```

Comment (AzmiTouil) [reply component/common/src/main/java/org/exoplatform/platform/common/space/rest/SpaceRestServiceImpl.java]: Done

**Pull request merged**